docs/jk_race_around_condition.md

```diff
@@ -6,6 +6,12 @@ nav_order: 14
 
 # Race around condition
```

Comment:
This heading should not be present in the TOC. Same goes for all other files that have the page title included in the heading.